dspace-api/src/test/java/org/dspace/core/ContextTest.java

```diff
@@ -255,6 +261,60 @@ public void testComplete2() throws SQLException {
         cleanupContext(instance);
     }
 
+    /**
+     * Test of commit method, of class Context.
+     */
+    @Test
+    public void testCommit() throws SQLException, AuthorizeException, IOException {
+        // To test commit() we need a new Context object
+        Context instance = new Context();
+
+        // By default, we should have a new DB connection, so let's make sure it is there
+        assertThat("HibernateDBConnection should exist", instance.getDBConnection(), notNullValue());
+        assertTrue("Context should be valid", instance.isValid());
+        assertTrue("Transaction should be open", instance.isTransactionAlive());
+
+        // Allow full Admin perms (in new context)
+        when(authorizeServiceSpy.isAdmin(instance)).thenReturn(true);
+
+        // Create a new EPerson (to be committed)
+        String createdEmail = "myfakeemail@gmail.com";
```

Comment:
Small point:  I'm always leery of testing with real email domains, even if the localpart "should be fake".  `myfakeemail@example.com` should be guaranteed undeliverable, since that domain is reserved.

Reply:
@mwoodiupui : Good point.  I've corrected this mistake (it was in two places in the same test) in this commit: https://github.com/DSpace/DSpace/pull/2674/commits/808316599e5e85ac0748c0b0cbc9c6f3e5990d40